units/XNL0301/XNL0301_script.lua

```diff
@@ -119,6 +125,8 @@ XNL0301 = Class(NCommandUnit) {
         end
     end,
 
+    ---@param self XNL0301
+    ---@return table
```

Comment:
```suggestion
    ---@return WeaponBlueprint
```

There is also a self:GetBlueprint() to replace below.